src/main/java/org/jabref/logic/importer/fetcher/isbntobibtex/BookInfoFetcher.java

```diff
@@ -0,0 +1,163 @@
+package org.jabref.logic.importer.fetcher.isbntobibtex;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+import org.jabref.logic.importer.AuthorListParser;
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.logic.importer.ParseException;
+import org.jabref.logic.importer.Parser;
+import org.jabref.logic.importer.fetcher.AbstractIsbnFetcher;
+import org.jabref.logic.importer.util.JsonReader;
+import org.jabref.logic.net.URLDownload;
+import org.jabref.model.entry.Author;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.Date;
+import org.jabref.model.entry.field.StandardField;
+import org.jabref.model.entry.types.StandardEntryType;
+import org.jabref.model.strings.StringUtil;
+
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONException;
+import kong.unirest.json.JSONObject;
+import org.apache.http.client.utils.URIBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+/**
+ * Fetcher for Book-Info.com.
+ * <a href="https://www.book-info.com/isbn/">book-info</a>.
+ */
+public class BookInfoFetcher extends AbstractIsbnFetcher{
+    private final String BASE_URL = "https://www.book-info.com/";
+    public BookInfoFetcher(ImportFormatPreferences importFormatPreferences) {
+        super(importFormatPreferences);
+    }
+
+    @Override
+    public URL getUrlForIdentifier(String identifier) throws URISyntaxException, MalformedURLException, FetcherException {
+        this.ensureThatIsbnIsValid(identifier);
+        return new URIBuilder(BASE_URL)
+                .setPathSegments("isbn", identifier)
+                .build()
+                .toURL();
+    }
+
+    @Override
+    public void doPostCleanup(BibEntry entry) {
```

Comment:
If you have an abstract base class you don't need to override methods that are equal to their super method